tests/integration/suite/test_cluster_defaults.py

```diff
@@ -96,3 +96,41 @@ def test_rke_k8s_deprecated_versions(admin_mc, remove_resource):
                                     'v1.8.10-rancher1-1 is deprecated'
     client.update_by_id_setting(id=deprecated_versions_setting.id,
                                 value="")
+
```

Comment:
@prachidamle do you think it would make sense to add a test for the action handler? To see if it gets disabled when templateId is set on the cluster

Reply:
done